Update docs for `Notifications for Android TV / Fire TV` for changes made to service data for icons and images (#31228)
home-assistant/home-assistant.github.io: source/_integrations/nfandroidtv.markdown

## Service data for sending images and icons

| Service data attribute | Optional | Description |
| ---------------------- | -------- | ----------- |

> | `image`                |      yes | Groups the attributes for image upload. It can take a `url` or `path` directly, however if you need to use any of the authentication options either `url` or `path` have to be provided. |

Reviewer: ```suggestion
| `image`                |      yes | Groups the attributes for image upload. It can take a `url` or `path`. It is optional unless you use authentication options. Then, either `url` or `path` has to be provided. |
```

> | `icon`                 |      yes | Groups the attributes for icon upload. It can take a `url` or `path` directly, however if you need to use any of the authentication options either `url` or `path` have to be provided. |

Reviewer: ```suggestion
| `icon`                 |      yes | Groups the attributes for icon upload.  It can take a `url` or `path`. It is optional unless you use authentication options. Then, either `url` or `path` has to be provided.  |
```

> | `path`                 |      yes | Local path of an image file. Is placed inside `image`, `icon` or both.

Reviewer: ```suggestion
| `path`                 |      yes | Local path of an image file. Is placed inside `image`, `icon`, or both.
```
